lib/business_time/currency.rb

```diff
@@ -0,0 +1,24 @@
+module BusinessTime::Currency
```

Comment:
```ruby
module BusinessTime
  module Currency
```

is safer in case when this file will be loaded _before_ any other, defining `BusinessTime` in rails env. That could be a case.